org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketServer.java

```diff
@@ -83,11 +83,11 @@ public void textMessage(String message) {
 
   @OnMessage
   public void byteMessage(ByteBuffer b) {
-    outputAdapter.sendMessage("Got a byte array message. Doing nothing.");
+    outputAdapter.sendMessage(new SystemOutMessage("Got a byte array message. Doing nothing."));
```

Comment:
should these be turned into keys?

Reply:
These are placeholders for now and will only show up on the localhost version. I'll leave them as is.